docs/settings.rst

```diff
@@ -31,9 +31,35 @@ click the *Promote to administrator* button.
 
 .. figure:: /_static/images/settings/user.png
 
+Replacements
+------------
+
 Administrators can also delete users,
 using the delete button next to the user name.
 
+If you want to delete a user account, for example,
+because the user is leaving your organization, unfinished tasks to this account may still exist.
+In this case, specify a replacement who will take over open task assignements.
+The substitute will then have exactly all access rights to complete these tasks,
+but not automatically inherits all rolls or rights of the deleted user.
```

Comment:
`rolls` should be `roles`
I'm a bit confused by the last part:
> but not automatically inherits all rolls or rights of the deleted user.

What exactly would be roles and rights?
It is correct that if you use roles within a case for tasks, the value there isn't changed. However, we adjust access rights in cases and tasks where necessary.

Another thing to mention would be the assignment for closed tasks is not changed for audit reasons.